armi/reactor/converters/axialExpansionChanger.py

```diff
@@ -818,14 +818,8 @@ def _isFuelLocked(self, b):
         - This serves as an example to check for fuel/clad locking/interaction found in SFRs.
         - A more realistic/physical implementation is reserved for ARMI plugin(s).
         """
-        c = b.getChildrenWithFlags(Flags.FUEL)
-        if len(c) == 0:  # pylint: disable=no-else-raise
-            raise RuntimeError("No fuel component within {0}!".format(b))
-        elif len(c) > 1:
-            raise RuntimeError(
-                "Cannot have more than one fuel component within {0}!".format(b)
-            )
-        self._componentDeterminesBlockHeight[c[0]] = True
+        c = b.getComponent(Flags.FUEL)
+        self._componentDeterminesBlockHeight[c] = True
```

Comment:
How does this prevent against issues that the previous implementation checked for? `getComponent` will return a `None` if there is no `FUEL` here. Will this work still or should we still check that?

Reply:
It won't. Good catch. I was going too fast and read the `runLog.warning` as a RuntimeError.
https://github.com/terrapower/armi/blob/e47abd2690e3833bf8e3ed02225df2ddea6bc017/armi/reactor/composites.py#L2477-L2483

I'll put that specific check back.